packages/runtime/test/integration/atomic-transfer.ts

```diff
@@ -75,8 +75,8 @@ describe("Algorand Smart Contracts - Atomic Transfers", function () {
         payFlags: { totalFee: 1000 }
       }
     ];
-    const initialJohnAssets = john.getAssetHolding(assetId)?.amount;
-    const initialAliceAssets = alice.getAssetHolding(assetId)?.amount;
+    const initialJohnAssets = john.getAssetHolding(assetId)?.amount as bigint;
+    const initialAliceAssets = alice.getAssetHolding(assetId)?.amount as bigint;
```

Comment:
same as above. Linter. I think i fixed this before, why was it reverted? :(

Reply:
It's also happened a lot of time with me. :(